packages/message-hash/src/index.ts

```diff
@@ -27,3 +27,12 @@ export function messageHash(
   }
   return sha256(bytes);
 }
+
+export function messageHashStr(
```

Comment:
I don't remember why this package was created but as I see it contains only one function.
Considering how it is used it can be safely moved to `core` and re-exported from `sdk`. 

@waku-org/js-waku-developers wdyt?

Reply:
Maybe because it depends on `@noble/hashes`. But `@waku/core` already relies on the same as well. So perhaps a good idea to move it into `@waku/core`.
PR here to move this: https://github.com/waku-org/js-waku/pull/1810 where we can also continue this discussion if required.